build-system/compile/compile.js

```diff
@@ -236,6 +236,8 @@ function compile(
     }
     externs.push('build-system/externs/amp.multipass.extern.js');
 
+    const es5Target = STRICT_EXPERIMENT ? 'ECMASCRIPT5_STRICT' : 'ECMASCRIPT5';
```

Comment:
Maintainability nit: This makes it two conditions for `language_out` on two lines (239 and 249). Can we consolidate the logic in one place? (Here is okay.)